README.md

```diff
@@ -31,6 +31,24 @@ Also:
 
 ## Anvil
 
+### Dependencies
+You'll need to install the following dependencies; note, that those package
+names may vary depending on your OS and linux distribution:
+- `libdbus-glib-1`
+- `libudev`
+- `libxkbcommon`
+- `libinput`
+- `libgbm`
```

Comment:
It would be nice to split out `libgbm`, `libudev` and `libinput` to separate section called `Udev / DRM backend`

Reply:
just edited it